### DISCUSSION TITLE
Unexpected Auto-capitalization behavior with Combo box

### DISCUSSION THREAD

**Discussion by SkylerLutz** [IgniteUI/igniteui-angular#12997] [Q&A] [answered]
## Description  
The country names in [this example](https://www.infragistics.com/products/ignite-ui-angular/angular/components/combo#two-way-binding) are auto-capitalized. Is there a way to disable the auto-capitalization behavior? Thanks

**Comment by RivaIvanova** [comment, marked as answer]
Hi @SkylerLutz 

I have been looking into your question and what I could say is that this behavior is by design and expected, however, the IgxCombo provides the [`igxComboHeaderItem`](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/IgxComboComponent.html#headerItemTemplate) directive that is used to template the header items for groups in the combo list.

For example, the following code will set a custom template where the header items are displayed in lowercase.

```html
<igx-combo>
    <ng-template igxComboHeaderItem let-item let-key="groupKey">
        <div class="header-item-class">{{ item[key] }}</div>
    </ng-template>
</igx-combo>
```
```scss
.header-item-class {
  text-transform: lowercase;
}
```

[Here](https://stackblitz.com/edit/bcgsh6?file=src%2Fapp%2Fcombo-template%2Fcombo-template.component.scss,src%2Fapp%2Fcombo-template%2Fcombo-template.component.html) could be found a small sample demonstrating this approach.
